eZ/Publish/API/Repository/Values/Content/Query/Criterion.php

```diff
@@ -146,6 +178,9 @@ private function getValueTypeCheckCallback($valueTypes)
         return $callback;
     }
 
+    /**
+     * @deprecated since 7.2, will be removed in 8.0. Use the constructor directly instead.
+     */
     public static function createFromQueryBuilder($target, $operator, $value)
     {
         return new static($target, $operator, $value);
```

Comment:
Let's also add the `@trigger_error('...' , E_USER_DEPRECATED)` as Symfony does.
(also in other similar places)